src/legacy/core_plugins/console/public/np_ready/application/models/legacy_core_editor/__tests__/input_tokenization.test.js

```diff
@@ -84,17 +73,15 @@ describe('Input Tokenization', () => {
     }
 
     test('Token test ' + testCount++ + ' prefix: ' + prefix, async function (done) {
-      input.update(data, function () {
-        const tokens = tokensAsList();
-        const normTokenList = [];
-        for (let i = 0; i < tokenList.length; i++) {
-          normTokenList.push({ type: tokenList[i++], value: tokenList[i] });
-        }
-
-        expect(tokens).toEqual(normTokenList);
-        done();
-      });
+      await coreEditor.setValue(data, true);
+      const tokens = tokensAsList();
+      const normTokenList = [];
+      for (let i = 0; i < tokenList.length; i++) {
+        normTokenList.push({ type: tokenList[i++], value: tokenList[i] });
+      }
 
+      expect(tokens).toEqual(normTokenList);
+      done();
```

Comment:
If you have an `async` test function then you don't need the `done` parameter ([docs](https://jestjs.io/docs/en/asynchronous#asyncawait)). I don't know whether you want to change that in this PR however.